modules/azurerm-virtual-network/variables.tf

```diff
@@ -0,0 +1,80 @@
+variable "address_space" {
+  description = "The address space that is used the virtual network. You can supply more than one address space."
+  type        = list(string)
+}
+
+variable "location" {
+  description = "The location/region where the virtual network is created. Changing this forces a new resource to be created."
+  type        = string
+}
+
+variable "name" {
+  description = "The name of the virtual network. Changing this forces a new resource to be created."
+  type        = string
+}
+
+variable "resource_group_name" {
+  description = "The name of the resource group in which to create the virtual network. Changing this forces a new resource to be created."
+  type        = string
+}
+
+variable "subnets" {
+  description = <<-EOF
+    [
+      {
+        address_prefixes: A list of address prefixes to use for the subnet
+        default_outbound_access_enabled: (Optional) Enable default outbound access to the internet for the subnet. Defaults to true.
+        name: The name of the subnet
+        route_table_id: (Optional) The ID of the Route Table that should be associated with this subnet.
+        security_group: (Optional) he Network Security Group to associate with the subnet. (Referenced by id, ie. azurerm_network_security_group.example.id)
+
+        deletegation: (Optional) One or more delegation blocks as defined below.
```

Comment:
typo in `delegation`

Reply:
this is marked as resolved but I still see the typo 👀

Reply:
I think github is being weird cause it's definitely updated in the [branch](https://github.com/imperva/terraform-dsfhub-agentless-onboarding/blob/matt/az-managed-instance/modules/azurerm-virtual-network/variables.tf#L31)